app/gui/view/graph-editor/src/component/visualization/container/action_bar.rs

```diff
@@ -389,7 +389,11 @@ impl ActionBar {
             hide              <- any(mouse_out_no_menu,remote_click);
             eval_ hide (model.hide());
 
-            frp.source.visualisation_selection <+ visualization_chooser.chosen_entry;
+            // The action bar does not allow to deselect the visualisation, so we prohibit these
+            // events, which can occur on re-initialization.
+            has_selection <- visualization_chooser.chosen_entry.is_some();
+            frp.source.visualisation_selection
+                <+ visualization_chooser.chosen_entry.gate(&has_selection);
```

Comment:
`.filter_map(|entry| *entry)` does it in one line. And I use it a lot - maybe we could make `filter_some` node?

Reply:
I think `.unwrap()` already does exactly that.